support/locust/test.json

```diff
@@ -1,14 +1,14 @@
 [
     {
-        "hostName": "http://172.17.0.1:8888",
+        "hostName": "http://clarity31.eecs.umich.edu:8080",
```

Comment:
Let's have the default here to be localhost.

Reply:
Resolved